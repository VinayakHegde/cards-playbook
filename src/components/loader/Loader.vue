<style lang="scss" scoped src="./style.scss"></style>
<template>
  <div class="overlay">
    <div class="loader"></div>
  </div>
</template>
