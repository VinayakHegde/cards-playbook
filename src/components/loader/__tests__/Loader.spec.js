import { mount } from "@vue/test-utils";
import Loader from "../Loader.vue";
describe("Loader.vue", () => {
  const wrapper = mount(Loader);
  it("match the snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
