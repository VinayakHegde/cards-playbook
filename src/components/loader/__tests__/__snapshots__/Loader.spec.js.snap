// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader.vue match the snapshot 1`] = `
<div class="overlay">
  <div class="loader"></div>
</div>
`;
